src/main.cpp

```diff
@@ -285,11 +283,10 @@ int main(int argc, char* argv[])
     QSharedPointer<QJackTrip> window;
 
 #ifndef NO_VS
-    QString deeplink = QLatin1String("");
+    QString deeplink;
     QSharedPointer<VirtualStudio> vs;
 #ifdef _WIN32
-    QSharedPointer<QLocalServer> instanceServer;
-    QSharedPointer<QLocalSocket> instanceCheckSocket;
+    QScopedPointer<VsInit> vsInit;
```

Comment:
This instance doesn't seem to be used but just reset for Windows. Could be removed here and declared later when used.